### PR TITLE
Update foreman_ansible to 5.0.0

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: plugins/ruby-foreman-ansible/debian/control [hunk 1]
```diff
@@ -2,18 +2,17 @@ Source: ruby-foreman-ansible
 Section: ruby
 Priority: extra
 Maintainer: Daniel Lobato Garcia <elobatocs@gmail.com>
-Build-Depends: debhelper, foreman (>= 1.24.0~rc1), ruby-foreman-deface (<<2.0.0),
-  foreman-assets(>= 1.24.0~rc1), ruby-foreman-tasks (>= 0.8.2-2), ruby-foreman-tasks (<<1.0.0),
-  ruby-foreman-remote-execution (>= 2.0.0-1), ruby-foreman-remote-execution (<<3.0.0),
-  foreman-sqlite3 (>= 1.24.0~rc1)
+Build-Depends: debhelper, foreman (>= 2.0.0~rc1), ruby-foreman-deface (<<2.0.0),
```
Comment (tbrisker) [inline plugins/ruby-foreman-ansible/debian/control]: Looks like this is failing because we don't have 2.0.0~rc1 yet

Comment (mmoll) [reply plugins/ruby-foreman-ansible/debian/control]: We have that in nightly. :)

File: plugins/ruby-foreman-ansible/debian/control [hunk 2]
```diff
@@ -2,18 +2,17 @@ Source: ruby-foreman-ansible
...
+  foreman-assets(>= 2.0.0~rc1), ruby-foreman-tasks (>= 0.8.2-2), ruby-foreman-tasks (<<1.0.0),
```
Comment (mmoll) [inline plugins/ruby-foreman-ansible/debian/control]: `ruby-foreman-tasks (<<1.0.0)` is the problem here.

Comment (ares) [reply plugins/ruby-foreman-ansible/debian/control]: oops, missed to drop this dependecy as well, will update in a sec

**Comment by mmoll** [comment]
merged, díky @ares!

**Pull request merged**